frontend/src/styles/mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin standard_button {
+  height: 50px;
+  background: transparent;
+  border-radius: 8px;
+  border: 1px solid $contrast-color;
```

Comment:
Perhaps the border-width should be bumped to `2px`. Monitors with low pixel density might get pixelated edges, having a slightly thicker border can help with this, and will make the buttons more visible on the page.

Reply:
✅